tensorboard/webapp/widgets/source_code/source_code_container_test.ts

```diff
@@ -17,7 +17,7 @@ limitations under the License.
  */
 import {Component, Input, NO_ERRORS_SCHEMA} from '@angular/core';
 import {ComponentFixture, TestBed} from '@angular/core/testing';
-import * as loadMonacoShim from './load_monaco_shim';
+import {MonacoShim} from './load_monaco_shim';
```

Comment:
This breaks the sync. Would you mind working on the fix? 
See cl/487331149. Thanks